corehq/apps/cloudcare/views.py

```diff
@@ -266,9 +277,11 @@ def _default_lang():
         # default language to user's preference, followed by
         # first app's default, followed by english
         language = request.couch_user.language or _default_lang()
+        domainobj = self._get_domain_obj(domain)
```

Comment:
Naming conventions: `domainobj` => `domain_obj` - although the same comment as above applies, I'd just update the line below to be `"default_geocoder_location": domainobj.Domain.get_by_name(domain)`.